lib/vsc/zk/rsync/source.py

```diff
@@ -312,10 +314,15 @@ def get_flags(self, file, recursive):
             flags.append('--delete')
         if self.rsync_checksum:
             flags.append('--checksum')
+        if self.rsync_hardlinks:
+            flags.append('--hard-links')
         if self.rsync_verbose:
             flags.append('--verbose')
         if self.rsync_dry:
             flags.append('-n')
+        if self.rsync_arbitopts:
```

Comment:
add a comment that this should always be processed last (e.g. in case new ones are added)